layouts/DocsPage/Versions.tsx

```diff
@@ -16,18 +18,42 @@ const Versions = ({
 }: VersionsInfo) => {
   const router = useRouter();
   const [currentItem, setCurrentItem] = useState<string>(current);
-
+  const [articleList, setArticleList] = useState<LinkWithRedirectList>({});
+  const currentPage = getCurrentPageWithScope(router.asPath);
   const versions = useMemo(() => [...available].reverse(), [available]);
 
+  useEffect(() => {
+    async function getRedirectsMap() {
+      const res = await require("../../utils/articleLinks.json");
+      setArticleList(res);
+    }
+
+    getRedirectsMap();
+  }, []);
+
   const navigateToVersion = useCallback(
     (version: string) => {
       const isLatest = version === latest;
```

Comment:
Q: Why we live part woth `isLatest` here and not move it inside `findExistingPage`? We will probably always need to make this check.